python/tvm/relay/op/strategy/arm_cpu.py

```diff
@@ -284,8 +288,21 @@ def conv2d_strategy_arm_cpu(attrs, inputs, out_type, target):
                     name="depthwise_conv2d_nchw.x86",
                 )
         elif layout == "NHWC":
-            assert kernel_layout == "HWOI"
-            if target.features.has_asimd:
+            # TODO(@FranklandJack)
+            # Handle HWOI in arm_cpu scheudles/compute definition.
```

Comment:
Nit:
```suggestion
            # Handle HWOI in arm_cpu schedules/compute definition.
```

Reply:
Haha whoops! I should turn my spell checker on, will fix :)